app/views/categories/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<% content_for :title, 'MoneyMate | Categories' %>
+  <%= javascript_import_module_tag "menu" %>
+
+<div class="grand-total">
+        <p class="amount-label">Total Amount</p>
+        <p class="amount"><%= "$#{@total}" %></p>
+</div>
+    <div class="container-fuild mx-2">
+    <% if @categories.present? %>
+    
+    <% @categories.each do |category| %>   
+       
+          <%= link_to  category_expenses_path(category), class: "card-link" do %>
+            <div class="card"> 
+              <div class="icon-container">
+                <%= image_tag(category.icon, class: "icon") %>
+              </div>
+          
+              <div class="text-container">
+                <div class="flex-row px-3">
+                  <h4 class="text-lg">
+                    <%= category.name %>
+                  </h4>
+                  <div class="amount">
+                    <%= "$#{category.total_amount}" %>
+                  </div>
+                </div>
+                <p class="text-xs px-2"><%= category.created_at.strftime("%d %b %Y") %></p>
+              </div>
+              
+            </div>
+          <% end %>
+          <% end %>
+    <% else %>
+        <div class="actions">
+            You don't have any category yet! Create one.
+        </div>
+    <% end %>
+
```

Comment:
- [ ] The category page(homepage) is messed-up, Showing categories images in big sizes that are overflowing with each other. I think It's because of the image resolution. Please make sure to fix this issue and your design should match the given design. I would suggest displaying categories images in fixed sizes, it should not rely on image resolution whatever size the user uses all images should display in a fixed size.

![image](https://user-images.githubusercontent.com/83803414/235747640-59e92603-5f77-459d-9598-e038b85d5feb.png)